dag/health.yml

```diff
@@ -62,7 +62,7 @@ steps:
 
   # Registered vehicles per 1000 people
```

Comment:
I think this has been archived in the grapher so we can actually remove it here too

Reply:
Good catch